lib/matplotlib/axes/_base.py

```diff
@@ -1363,6 +1363,18 @@ def __getitem__(self, key):
                     for artist in self._axes._children
                     if self._type_check(artist)][key]
 
+        def __add__(self, other):
```

Comment:
both `if`s can be compressed to `if isinstance(other, (list, ArtistList)): return [*self, *other]`?

Reply:
Yes.